lib/build-database.js

```diff
@@ -321,23 +322,31 @@ const importLines = async trx => {
           Object.entries( gurmukhi ).forEach( ( [ sourceName, gurmukhi ] ) => {
             const sourceId = sourceNames.findIndex( nameEnglish => sourceName === nameEnglish ) + 1
 
-            lineContentData.push( {
-              line_id: line.id,
-              source_id: sourceId,
-              gurmukhi,
+            const getFirstLetters = text => ( {
               first_letters: [
+                toUnicode,
                 stripEndings,
```

Comment:
https://github.com/ShabadOS/gurmukhi-utils/pull/163

Check if line is `Sirlekh`